### ISSUE TITLE
linter-markdown not respecting .remarkrc

### ISSUE THREAD

**Issue by nheath** [AtomLinter/linter-markdown#102]
I installed linter-markdown, looking for a replacement to https://atom.io/packages/tidy-markdown, so that i could customize an rc file, but so far I have not been able to get this to work. I created a few sample files showing the issues I am seeing.

.remarkrc
```json
{
  "presets": [
    "remark-preset-lint-recommended"
  ],
  "plugins": {
    "lint": {
      "unordered-list-marker-style": "*",
      "table-pipes": false,
      "table-pipe-alignment": false,
      "list-item-indent": "space"
    }
  }
}
```

test.md
```markdown
| test | test |
| ---- | ----
foo  | bar

a list

- first

```
![screen shot 2016-08-23 at 8 44 36 pm](https://cloud.githubusercontent.com/assets/1778613/17917556/6d05928c-6972-11e6-9481-4b8032d64fc6.png)

I know that remark is running, obviously, and I can also see that its seeing my .remarkrc file, because if it contains malformed json, i get an error. It looks like it's just ignoring the settings. I have also tried moving the .remarkrc file to my home directory, and can similarly see that atom is picking up the file there. Am I doing something wrong? It looks like this is a duplicate of #62.

Second, this is probably a different issue, belonging somewhere else, but i am a bit confused by which remark projects are responsible for what. The documentation mentions support for automatically fixing markdown; this is similar to tidy-markdown, which formats your markdown when you save the document. Is it possible in linter-markdown or linter-remark, to automatically fix markdown during document save in atom?

**Comment by wooorm** [comment]
Hi @nheath, thanks for opening an issue!

remark/remark-lint just received huge updates, which aren’t yet in linter-markdown or linter-remark.

You can install `remark-lint` in the project you’re working on, though, in which case this linter uses that instead of the built-in one.

> Is it possible in linter-markdown or linter-remark, to automatically fix markdown during document save in atom?

No, neither can do that. [atom-beautify](https://github.com/Glavin001/atom-beautify) comes with a prettifier built on remark though (although there’s one style, and no support for config files from what I gather).

I also think it’s rather outside of the scope for a “linter” to reformat, but @Arcanemagus can probably answer that better.

I’ll open another issue on how to bring the next version of remark/remark-lint to linter-markdown.

**Comment by nheath** [comment]
Thanks, I understand about reformatting being out of scope. I wouldn't have assumed that was a linter's job, but it was an advertised feature of remark-lint in the readme, so I thought I'd ask to see if there were plans to integrate this into the atom plugin. 

FWIW, I did try installing the latest version of remark-lint and **all** lint messages went away, but based on #103, it looks like this is the expected behavior for this version of the plugin with the latest remark-lint. I'll track that issue for further updates regarding the update.

Given that context, this issue may still be relevant to the current version of linter-markdown, as it does not appear to respect any rc files. I have disabled the plugin until either this or #103 is released. If the plan is to fix-forward, instead of the current release, this can be closed in favor of #103.

Thanks.
